test/unit/test_server.py

```diff
@@ -102,6 +102,18 @@ def check_api_compatible(name):
         yield check_api_compatible, name
 
 
+def test_setup_with_sproxyd_endpoints():
+    endpoint1 = 'http://h1:81/p'
+    endpoint2 = 'http://h2:81/p'
+    conf = {'sproxyd_endpoints': ' , '.join([endpoint1, endpoint2])}
+    obj_serv = swift_scality_backend.server.app_factory(conf)
+
+    assert frozenset([
+        urlparse.urlparse(endpoint1),
+        urlparse.urlparse(endpoint2),
+    ]) == obj_serv._get_client_for_policy(0).read_clients[0]._endpoints
+
+
```

Comment:
I think it makes sense to add an additional test which asserts failure on attempt of configuration with malformed urls.